openwisp_monitoring/device/api/views.py

```diff
@@ -194,6 +195,61 @@ def _write(self, request, pk):
         ct = ContentType.objects.get_for_model(Device)
         for interface in data.get('interfaces', []):
             ifname = interface['name']
+            if 'mobile' in interface:
```

Comment:
can you try to move this new block of code to its own method like we do with `_write_cpu`, `_write_disk` and `_write_memory`?

Reply:
Done